nilearn/mass_univariate/tests/test_permuted_least_squares.py

```diff
@@ -36,12 +36,12 @@ def get_tvalue_with_alternative_library(tested_vars, target_vars, covars=None):
 
     Returns
     -------
-    t-values: np.ndarray, shape=(n_regressors, n_targets)
+    t-values: np.ndarray, shape=(n_regressors, n_descriptors)
 
     """
     ### set up design
     n_samples, n_regressors = tested_vars.shape
-    n_targets = target_vars.shape[1]
+    n_descriptors = target_vars.shape[1]
```

Comment:
Hi, Can you explain the motivation for this change ?

Reply:
This is what it's called in the `permuted_ols` docstring. I kept having to refer to the docstring when going through the tests because of the variable terminology, so I figured it would be easier to read if everything was as consistent as possible between the function docstring and the tests.